### ISSUE TITLE
Antlines and signage don't transfer to the front of Flat and Angled Blocks

### ISSUE THREAD

**Issue by FrozenDragon0** [BEEmod/BEE2-items#3051]
**BEE2 version:** 4.39.2

**Description of the bug:** Antlines and signage don't transfer to the front of flat or angled blocks.

**Expected behavior:** Per the description of flat blocks and angled blocks, setting "Start Enabled" to true should make antlines and signage transfer to the front of the blocks. For angled blocks, I would expect this means the antline goes across the sloped surface. This doesn't currently work for either--the antlines stay on the wall behind the blocks regardless of whether "Start Enabled" is true or false.

**Steps to reproduce:** Have antlines go behind a flat or angled block, set the block to "Start Enabled" true.

**Additional information:**
Both the flat block and angled block in the screenshot have "Start Enabled" set to true:

![image](https://user-images.githubusercontent.com/83612383/119241166-89327680-bb22-11eb-8479-b356d43bf364.png)

![image](https://user-images.githubusercontent.com/83612383/119241171-92234800-bb22-11eb-9562-946cb8527aea.png)

![20190710154803_1](https://user-images.githubusercontent.com/18079316/60950803-6f228d00-a32a-11e9-90ce-a80a4891c38d.jpg)

**Comment by Libbybapa** [comment]
The antlines on the voxel the block is stuck to are transferred to the diagonal surface.  Is that not what you are experiencing?  It works fine for me.  You have 'start enabled' checked?

**Comment by Libbybapa** [comment]
I don't think this is a bug.  I think it is a lack of understanding of how the system works.

**Comment by Lieutenant-L-T-Smash** [comment]
I confirm this is an problem in BEE 4.39.2.

Neither antlines nor signage transfer to the face of the block. It does not matter how it is placed.

**Comment by LincoCompany00** [comment]
This still isn't working in 2.4.45
